sound/soc/intel/boards/bytcr_rt5640.c

```diff
@@ -960,7 +963,18 @@ static int byt_rt5640_add_codec_device_props(const char *i2c_dev_name)
 	if (byt_rt5640_quirk & BYT_RT5640_JD_NOT_INV)
 		props[cnt++] = PROPERTY_ENTRY_BOOL("realtek,jack-detect-not-inverted");
 
-	ret = device_add_properties(i2c_dev, props);
+	fwnode = fwnode_create_software_node(props, NULL);
+	if (IS_ERR(fwnode)) {
+		/* put_device() is not handled in caller */
+		put_device(i2c_dev);
```

Comment:
would it be desirable to unify this with bytcr_rt5651? If you decide to add that to this PR, you might as well remove the spurious empty line added at the end of `snd_byt_cht_es8316_mc_remove()` above

Reply:
@lyakh the two machine drivers handle properties in different ways, I am not sure if they can be unified. That would be for @jwrdegoede to check, the code came from his contributions.

Will remove the spurious newline.

Reply:
Sorry, I didn't mean unifying the two drivers completely, I just meant the handling of `put_device()` - either within or outside of `byt_rt5651_add_codec_device_props()` and `byt_rt5640_add_codec_device_props()` respectively

Reply:
The structure is rather different so I am not sure if we can unify. The functions have different prototypes so it's not small surgery I am afraid.